tensorflow/python/ops/image_ops_impl.py

```diff
@@ -407,7 +407,7 @@ def _random_flip(image, flip_index, seed, scope_name):
   Args:
     image: 4-D Tensor of shape `[batch, height, width, channels]` or 3-D Tensor
       of shape `[height, width, channels]`.
-    flip_index: Dimension along which to flip image. Vertical: 0, Horizontal: 1
+    flip_index: Dimension along which to flip the image. Vertical: 0, Horizontal: 1
```

Comment:
Please make sure to not pass 80 characters limit on a line. Make sure you run pylint

Reply:
made required changes